<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>medico_expediente_clinico</class>
 <widget class="QWidget" name="medico_expediente_clinico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="btn_informacion">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #3E64FF;
border: none;
color: white;</string>
   </property>
   <property name="text">
    <string>Información General</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_alergias">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #3E64FF;
border: none;
color: white;
</string>
   </property>
   <property name="text">
    <string>Alergias</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_enfermedad">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #3E64FF;
border: none;
color: white;</string>
   </property>
   <property name="text">
    <string>Enfermedades</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_operacion">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #3E64FF;
border: none;
color: white;</string>
   </property>
   <property name="text">
    <string>Operaciones</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>481</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="lbl_imagen">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources.qrc">:/imagenes/Imagenes/imgPrueba.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_nombre">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>María Fernanda Cruz Zamora</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Estatura:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Peso:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tipo de Sangre:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>390</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Descripción:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_tipoSangre">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ORH +</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_estatura">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>290</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1.62</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_peso">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>340</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>62</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txt_descripcion">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>420</y>
       <width>421</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_modificarDatos">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>550</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
     </property>
     <property name="text">
      <string>Modificar Datos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_GenerarExpediente">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>550</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
     </property>
     <property name="text">
      <string>Generar Expediente</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="lbl_imagen_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>131</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../resources.qrc">:/imagenes/Imagenes/imgPrueba.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_nombre_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>María Fernanda Cruz Zamora</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Estatura:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tipo de Sangre:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>390</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Descripción:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txt_descripcion_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>420</y>
       <width>421</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>340</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Peso:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_tipoSangre">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>ORH +</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_estatura">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>290</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>1.62 </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_peso">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>340</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>62 </string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_guardarCambios">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>540</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
     </property>
     <property name="text">
      <string>Guardar Cambios</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Cancelar_modificar_expediente">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>540</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: red;
	border: 1px solid red;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: red;
}</string>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #818181;
background:none;</string>
     </property>
     <property name="text">
      <string>Alergias</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableView" name="table_alergias">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>100</y>
       <width>291</width>
       <height>371</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>291</width>
       <height>371</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>291</width>
       <height>371</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="btn_agregar_alergia">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>530</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
     </property>
     <property name="text">
      <string>Agregar Alergia</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_eliminar_alergia">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>530</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: red;
	border: 1px solid red;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: red;
}</string>
     </property>
     <property name="text">
      <string>Eliminar Alergia</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="btn_agregar_enfermedad">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>530</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
     </property>
     <property name="text">
      <string>Agregar Enfermedad</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_enfermedades">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>291</width>
       <height>341</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>291</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #818181;
background:none;</string>
     </property>
     <property name="text">
      <string>Enfermedades Padecidas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_eliminar_enfermedad">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>530</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: red;
	border: 1px solid red;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: red;
}</string>
     </property>
     <property name="text">
      <string>Eliminar Enfermedad</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QPushButton" name="btn_agregar_operacion">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>530</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: #3E64FF;
}</string>
     </property>
     <property name="text">
      <string>Agregar Operación</string>
     </property>
    </widget>
    <widget class="QTableView" name="table_operaciones">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>291</width>
       <height>341</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>291</width>
       <height>341</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Montserrat Medium</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #818181;
background:none;</string>
     </property>
     <property name="text">
      <string>Operaciones</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_eliminar_operacion">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>530</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Montserrat</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
	background-color: white;
	color: red;
	border: 1px solid red;
	border-radius: 5px;
	padding: 5px;
}

:hover{
	color: white;
	background-color: red;
}</string>
     </property>
     <property name="text">
      <string>Eliminar Operación</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #3E64FF;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>btn_informacion</zorder>
  <zorder>btn_alergias</zorder>
  <zorder>btn_enfermedad</zorder>
  <zorder>btn_operacion</zorder>
  <zorder>stackedWidget</zorder>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
